sound-design.MD

```diff
@@ -0,0 +1,42 @@
+# MRE Sound API proposal
+
+This is node.js side only. It doesn't actually do anything yet.
+
+## Required functionality
+* preload sounds (as looping or one-shot)
+* play preloaded sounds
+* load and play sounds (Basically preload, instantly play, unload upon completion)
```

Comment:
>load and play sounds [](start = 2, length = 20)

I wonder if the latency of loading the sound would nullify the utility of this method. Don't most sounds require timing?

Reply:
I think this may be a useful utility sound for some simple things, but not the method that gives the most control.It's the equivalent of CreateFromGLTF vs instantiating preloaded GLTF, I think it's useful.